### ISSUE TITLE
Theres No mods button on the menu

### ISSUE THREAD

**Issue by swirlyboii** [Temperz87/ultra-mod-manager#40]
Soo when i was just trying to **install** UMM it didint show the mods button on the the menu

**Comment by swirlyboii** [comment]
heres a pic:
![image](https://github.com/Temperz87/ultra-mod-manager/assets/161848764/cb5cd10b-c8c5-4a5e-b26c-2706af179ec2)

**Comment by IWannaModGames** [comment]
Bro i have the **same issue**

**Comment by kevbealive** [comment]
/issues/34

**Comment by rationalll** [comment]
same shit happening to me

**Comment by swirlyboii** [comment]
i think its bcz of the new update

**Comment by kierkat10** [comment]
Yes swirlyboii, it is. Just wait a few more... weeks? months? until a new version of UMM comes out...






...if it ever does.

**Comment by IWannaModGames** [comment]
I just realized this new version of UMM is **_deprecated, and no longer works._** you have to wait until the **_new version of UMM comes out_**

**Comment by Temperz87** [comment]
At this point I'm considering just archiving the repository.

1. I don't really care for Ultrakill modding that much.
2. UMM's implementation made it such that no modder wanted to use UKAPI, so it might be better to just split off UMM and UKAPI, however I'm not sure if an in game mod manager is even needed given the current reliance on thunderstore for the game's mod and the existence of r2mm
3. I don't understand why people want UMM back, and why it's particularly non-modders as well. 

If someone wants to update the manager go ahead, UMM is licensed under the MIT license, but given that no one has done so yet for a "popular" mod, I don't see why this manager needs updating.
